src/bindings/python/tests/test_graph/test_manager.py

```diff
@@ -177,7 +177,7 @@ def test_serialize_pass_tuple(request, tmp_path):
     model = ov.floor(ov.minimum(ov.abs(parameter_a), ov.multiply(parameter_b, parameter_c)))
     func = Model(model, [parameter_a, parameter_b, parameter_c], "Model")
     pass_manager = Manager()
-    pass_manager.register_pass("Serialize", output_files=(str(xml_path), str(bin_path)))
+    pass_manager.register_pass("Serialize", output_files=(xml_path, bin_path))
```

Comment:
yes, I found out that in my prev PR I make a mistake in `create_filename_for_test` function, so it always returned `Path` object -> I made changes in this test, but it can work only with strings.